metaflow/plugins/airflow/airflow.py

```diff
@@ -349,8 +349,8 @@ def _to_job(self, node):
             "METAFLOW_CODE_URL": self.code_package_url,
             "METAFLOW_CODE_DS": self.flow_datastore.TYPE,
             "METAFLOW_USER": user,
-            "METAFLOW_SERVICE_URL": BATCH_METADATA_SERVICE_URL,
-            "METAFLOW_SERVICE_HEADERS": json.dumps(BATCH_METADATA_SERVICE_HEADERS),
+            "METAFLOW_SERVICE_URL": SERVICE_URL,
+            "METAFLOW_SERVICE_HEADERS": json.dumps(SERVICE_HEADERS),
```

Comment:
I was thinking that why not call that `config_values` function here so that most of these can be removed directly and any new integration won't require changing multiple places. If that makes sense, I can throw a PR to add that here and a few other places.

Reply:
That was what my first version did. I removed that to reduce the testing area. I am perfectly happy if you want to do that but to get this PR through quickly, I would rather that be a separate PR that you guys own and test out.